WordPress/Classes/Models/ReaderSiteTopic+Lookup.swift

```diff
@@ -34,8 +34,12 @@ extension ReaderSiteTopic {
     ///
     /// - Parameter feedID: The feed id of the topic
     @objc(lookupWithFeedID:inContext:)
-    static func objc_lookup(withFeedID feedID: NSNumber, in context: NSManagedObjectContext) -> ReaderSiteTopic? {
-        try? lookup(withFeedID: feedID, in: context)
+    static func objc_lookup(withFeedID feedID: NSNumber?, in context: NSManagedObjectContext) -> ReaderSiteTopic? {
```

Comment:
This was interesting to me.

When `feedID` was not an optional, it could still be an uninitialized `NSNumber`. In Swift, `feedID == nil` would output `true` in this case. Xcode would show a warning that the non-optional `feedID` would never be `nil` and that check would always return `false`, but of course that wasn't accurate.

By making it an optional I think we can trust that the value with either be `nil` or an initialized `NSNumber`.

Reply:
I think part of the problem is that `ReaderPost.h` doesn't have any nullability identifiers for its properties. 

If we generate Swift interface, `feedID` would be `NSNumber!` by default. 

We could make it nullable:

```
@property (nonatomic, strong, nullable) NSNumber *feedID;
```

However, that would not solve the problem, since we call `objc_lookup` from `Obj-C`, so compiler would still allow calling an old version of this method with `NSNumber` rather than `NSNumber?`, so all the changes you made are necessary.